delphi/app/routes.py

```diff
@@ -6,11 +6,11 @@
 def updateSensorData():
     data = request.get_json(silent=True)
 
-    if data:
+    if isinstance(data, dict):
```

Comment:
request.get_json returns a dictionary (funny enough the documentation doesn't say, but that is what it does), so this check is not needed

Reply:
also the request body will be json/xml/something else that's part of the http spec so this check can hurt more than it helps